app/data/jsonSchemaForm/fundingAgreementSchema.ts

```diff
@@ -43,6 +43,10 @@ export const fundingAgreementSchema = {
       title: "Project Assets Life End Date",
       type: "string",
     },
+    proponentsSharePercentage: {
```

Comment:
@BCerki, check if there's something special we have to do to change schemas in prod  (`schema/data/prod/json_schema/funding_agreement.json`)